integration-tests/avs_test_generator.go

```diff
@@ -31,8 +31,8 @@ type AvsConfig[Input any, Output any] struct {
 	EthHttpUrl string
 	EthWsUrl   string
 
-	// The calculator to respond the tasks
-	ResponseCalculator operator.ResponseCalculator[Input, Output]
+	// A function that creates a calculator to respond the tasks
+	NewResponseCalculatorFn func() operator.ResponseCalculator[Input, Output]
```

Comment:
I think "Builder" might better convey the meaning

```suggestion
	ResponseCalculatorBuilder func() operator.ResponseCalculator[Input, Output]
```

Reply:
Done in [49371c7](https://github.com/Layr-Labs/eigensdk-go/pull/730/commits/49371c76f5ea6d7fdb2c2a47c4328d372b1d01ab)